app.py

```
@@ -3,7 +3,7 @@
 import pickle
 
 DB_USER = "admin"
-DB_PASS = "SuperSecre2t123456711111!11"
+DB_PASS = "SuperSecre2t123456711111!111"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "DB_PASS"

(S105)